Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -136,6 +121,22 @@ void continueAction(Task t)
             }
         }
 
+        private void RegisterResultsUpdatedEvent()
+        {
+            foreach (var pair in PluginManager.GetPluginsForInterface<IResultUpdated>())
+            {
+                var plugin = (IResultUpdated)pair.Plugin;
+                plugin.ResultsUpdated += (s, e) =>
+                {
+                    if (e.Query.RawQuery == QueryText) // TODO: allow cancellation
+                    {
+                        PluginManager.UpdatePluginMetadata(e.Results, pair.Metadata, e.Query);
+                        _resultsUpdateQueue.Post(new ResultsForUpdate(e.Results, pair.Metadata, e.Query, _updateToken));
+                    }
+                };
+            }
+        }
+
```

Comment:
what's this change?

Reply:
That's due to unintended merge. It seems just swap the posible of the two registration.

Reply:
There is a bit of difference, can you please double check including it is ok